utils/chart.ts

```diff
@@ -43,12 +45,29 @@ const getMedianDetails = (dataset: number[]): MedianDetails => {
   }
 }
 
-export const getChartData = (data: ChartData = []): RenderedChartData =>
+export const getChartPoints = (data: ChartData = []): Point[] =>
   data.map((item) => ({
-    x: item[0],
+    x: item[0].getTime(),
     y: item[1],
   }))
 
+export const getChartDataByTimeRange = (data: ChartData, timeRange: number) => {
+  if (!data) {
+    return
+  }
+  if (timeRange === 0) {
+    return data
+  } else {
+    const now = new Date()
+    const startDate = new Date(
+      now.getFullYear(),
+      now.getMonth(),
+      now.getDate() - timeRange,
+    )
```

Comment:
It's wrong. Please check this example:
<img width="174" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/4aca319c-f979-428e-b94d-a474e2c60817">

Reply:
Weird, because it's the same value as before my changes

Reply:
If you look to original code I just used the parameter instead ref variable, but it should be the same
![image](https://github.com/kodadot/nft-gallery/assets/64609060/60d36f07-11e9-4698-acb2-499d16951110)